subsys/modbus/Kconfig

```diff
@@ -47,6 +47,13 @@ config MODBUS_SERIAL
 	imply UART_USE_RUNTIME_CONFIGURE
 	help
 	  Enable Modbus over serial line support.
+	  
+config MODBUS_SERIAL_ASYNC_API
+	depends on MODBUS_SERIAL
+    bool "Modbus over serial line with async api support"
+    default n
```

Comment:
It also depends on `UART_ASYNC_API`, and `default n` is the default so it can be left out :)

Reply:
Thanks for the feedback. Changes are done.